lib/pbench/server/database/models/tracker.py

```diff
@@ -648,12 +675,13 @@ class Metadata(Database.Base):
     REINDEX = "REINDEX"
     ARCHIVED = "ARCHIVED"
     TARBALL_PATH = "TARBALL_PATH"
+    INDEX_MAP = "INDEX_MAP"
```

Comment:
Can you explain what `INDEX_MAP` mean?

Reply:
It's a record of all of the indices and document IDs associated with the dataset, so we can easily find them again for publish, delete, and other bulk Elasticsearch operations.

This is just defining the standard keys, and I never thought of trying to add an explanation here, but adding a comment for each key might not be a bad idea.